docs/intro/ff-ui/builder.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: builder
+title: App Builder
+tags: []
+---
+
+# App Builder
+
+On opening the project, you'll see the App Builder, which consists of four main sections: Navigation Menu, Toolbar, Canvas, and Properties panel.
+
+![AppBuilder.avif](imgs/AppBuilder.avif)
+
+## 1. Navigation Menu
+
+The [Navigation Menu](navigation-menu) provides access to the project's widget palette, widget tree, storyboard, and features like Firestore, API calls, asset uploads, app values, integrations, custom functions, theme settings, and more.
+
+## 2. Tool Bar
+
+From [Tool Bar](toolbar), you can search for project resources, change canvas size, see project history, branching, optimization and enhancements, view-download code, and run your app.
+
+## 3. Canvas Area
+
+In the [Canvas Area](canvas), you can see a preview of your mobile device's screen and build your app page.
+
+## 4. Properties Panel
+
+The Properties Panel lets you modify both the visual appearance and interactive behavior of UI elements on the canvas. It allows you to add Actions, set up a Backend Query, add Animations and more.
```

Comment:
Is there no page for Properties Panel?

Reply:
I had added it earlier, but since we are not talking about specific widgets or page properties, I just kept the brief here.